backend/pkg/elasticsearch/client/augur_client_queries.go

```diff
@@ -107,12 +107,12 @@ type SearchAfterParams struct {
 
 type SearchAfterResult struct {
 	Success *SearchAfterSuccess
-	Error   *error
+	Error   error
 }
 
 type SearchAfterSuccess struct {
-	CreatedAt time.Time
-	Result    []map[string]interface{}
+	ContinueParams SearchAfterParams
```

Comment:
Did this to encapsulate the CreatedAt within the struct, so things using SearchAfter don't have to care. Then again, I need a copy function to make it work completely.